<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_add_entry</class>
 <widget class="QWidget" name="Qmain_add_entry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="Qhoriz_layout_main" stretch="2,4,1">
     <item>
      <layout class="QVBoxLayout" name="Qvert_layout_actions" stretch="1,1,0,3,0,1">
       <item>
        <widget class="QPushButton" name="Qbutton_save">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Salveaza</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Qbutton_abort">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Renunta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Qlabel_obs">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Observatii:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Qtext_obs"/>
       </item>
       <item>
        <widget class="QLabel" name="Qlabel_error">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Erori:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Qtext_error">
         <property name="cursor" stdset="0">
          <cursorShape>ForbiddenCursor</cursorShape>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Qvert_layout_boxes">
       <item>
        <layout class="QVBoxLayout" name="Qlayout_name_p" stretch="0,0">
         <item>
          <widget class="QLabel" name="Qlabel_name_p">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nume persoana:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_name_p"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_dates">
         <item>
          <widget class="QLabel" name="Qlabel_dates">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Date persoana:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_dates"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_name_f">
         <item>
          <widget class="QLabel" name="Qlabel_name_f">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nume firma:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_name_f"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_cui">
         <item>
          <widget class="QLabel" name="Qlabel_cui">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CUI firma:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_cui"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_email">
         <item>
          <widget class="QLabel" name="Qlabel_email">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>E-mail firma:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_email"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_tel">
         <item>
          <widget class="QLabel" name="Qlabel_tel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Telefon firma:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_tel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_num_a">
         <item>
          <widget class="QLabel" name="Qlabel_num_a">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Numar angajati:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_num_a"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_caen">
         <item>
          <widget class="QLabel" name="Qlabel_caen">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Cod CAEN:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_caen"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Qlayout_fund">
         <item>
          <widget class="QLabel" name="Qlabel_fund">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ce se doreste a fi finantat:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Qtext_fund"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="Qvert_layout_reminder" stretch="0,3,0,1">
       <item>
        <widget class="QLabel" name="Qlabel_act">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Actiuni firma:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Qtext_act">
         <property name="cursor" stdset="0">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Qlabel_reminder">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reminder:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="Qtext_reminder">
         <property name="cursor" stdset="0">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
